File: week11/week11-1.py
#week11-1.py 主題 Hash Map Hash Set
table = {} #python 用大括號 代表字典 也就是 Hash table(Hash Map Hash Set)
table[9977341] = 9
table[4433997] = 4

print(table[9977341]) # 會印出 9
print(table[4433997]) # 會印出 4

if 0 not in table:
  print("table[0]不存在")

File: week11/week11-1.cpp
/// Hash Map or Hash Set
/// 不能直接在 codeblocks 用 因為是 C++2021
/// setting-complier 嫁入 C++11 就可以
#include <iostream>
#include <unordered_map>
using namespace std;

int main(){
    unordered_map<int,int> table;
    table[9977341] = 9;
    table[4433997] = 4;
    cout << table[9977341] << "\n";
    cout << table[4433997] << "\n";
}
